src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1563,6 +1563,7 @@ class C
                 state.CompletionCommandHandler.ExecuteCommand(New DeleteKeyCommandArgs(view, state.SubjectBuffer), Sub() editorOperations.Delete())
 
                 state.AssertNoCompletionSession()
+                view.Close()
```

Comment:
Maybe wrap the previous few lines in try/finally to ensure this always gets called?